### DISCUSSION TITLE
How use consoletvs/charts in domPdf to create charts?

### DISCUSSION THREAD

**Discussion by JorgeLavao** [dompdf/dompdf#3286] [General]
I am trying to create a pdf with graphs using the consoletvs/charts library with the charts.js cdn but it does not show the graphs, please help

**Comment by bsweeney** [comment]
I don't think this will work with Dompdf. I'm not familiar with either library, but it looks like consoletvs/charts generates the JavaScript necessary to display a chart using chart.js. Since Dompdf doesn't support JavaScript at all you'd need a way to render the chart on the server side in an image format (PNG, WebP, SVG).
